test/integration/router_test.go

```diff
@@ -356,12 +354,6 @@ func TestRouter(t *testing.T) {
 		if err := waitForRoute(tc.routerUrl, tc.routeAlias, tc.protocol, nil, tc.expectedResponse); err != nil {
 			t.Errorf("TC %s failed: %v", tc.name, err)
 
-			// The following is related to the workaround above, q.v.
-			if getRouterImage() != defaultRouterImage {
```

Comment:
@kargakis  quite frankly I can't recall why that was put in. I think it was probably because some custom image tests failed when tests were run with dind (docker in docker).